cfme/tests/containers/test_overview_data_integrity.py

```diff
@@ -82,3 +82,35 @@ def test_containers_overview_data_integrity(appliance, soft_assert):
                 cls.__name__, api_values[cls], statusbox_value
             )
         )
+
+
+@pytest.mark.manual
+@pytest.mark.tier(1)
```

Comment:
For this file we already have global `tire(1)`

Reply:
Good catch. I will fix that